### ISSUE TITLE
DOCUMENTATION - CREATE READ ME

### ISSUE THREAD

**Issue by keanacobarde** [keanacobarde/on-paper#16]
# User Story
I, as the curious employer, or the would-be contributor want to see the process of the creation of this application. This is best and most immediately seen within the readme. As a curious employer, I should get an immediate summary of what the project is about and have plenty of images that display the process and NOT just paragraphs of documentation. That's for second viewing!

# Acceptance Criteria
SUGGESTED TEMPLATE. I know there's an example of a capstone read me somewhere. It'll be attached when I can find it:
```
Name of the Project
Overview of the project
Link to your wireframes/prototype
Link to the deployed project
Link to your project board (yes...even though it is a part of the repo)
Description of the user
List of features
Screenshots of your project
List of contributors and links to their GH profiles
Link to Loom video walkthrough of your app (no more than 1 minute long! Make it great)
```

# Dependecies
N / A - This is an issue that can be tackled as the project goes along. All of your diagrams should be attached. Screenshots of the completed product should be attached. Your wireframe should be attached.

# Dev Notes
- Writing in markdown.
- Use past readme templates. 

SAMPLE FROM HHPW
```
# HIP HOP PIZZA AND WANGS [![Netlify Status](https://api.netlify.com/api/v1/badges/0a1d8099-ac63-4f29-84f0-c4f2225482e8/deploy-status)](https://app.netlify.com/sites/hiphoppizzaandwangspos/deploys)
<!-- update the netlify badge above with your own badge that you can find at netlify under settings/general#status-badges -->

Do you like pizza? Do you like wings? Do you like Hip-Hop? Why have one when you can have all three! Hip Hop Pizza and Wangs is your one stop shop for it all! You have your pizza, you have your wings, but how do you pay for it all? No fear! Our point of sales system is here. 

[View App](#https://hiphoppizzaandwangspos.netlify.app/)

## Get Started <!-- OPTIONAL, but doesn't hurt -->
First, clone this repo. Pick your favorite text editor. In your terminal, enter the following:

```
npm start
```
This will stand the live server. Authenticate with your email - pizza picking paradise awaits.

## About the User <!-- This is a scaled down user persona -->
- We were comissioned by the high lords of hip hop - the owners of HHPW - to create this application. Ideally, employees of this fine establishment authenticate through FireBase Oauth, and utilize this system to create/edit incoming orders, add items to said orders, close orders, and catch a peek at the current revenue of a set date range. Payment types and tips are all accounted for.

## Features <!-- List your app features using bullets! Do NOT use a paragraph. No one will read that! -->
- A CRUD application featuring persistent storage within a FireBase DB. 
- Authenticated users can log in and CREATE orders.
- Authenticated users can READ created orders as well as items associated with them.
- Authenticated users can UPDATE orders. 
- Authenticated users (AUs) can DELETE orders as well as items associated with orders.
- AUs can choose from a list of pre-determined items on a menu to add to orders.
- AUs can see a revenue page that includes a summary of the following: total revenue, total tips, total call ins, total walk ins, as well as the number of each payment types.

## LOOMS OF HHPW<!-- A loom link is sufficient -->
*Click below to see HHP+WPOS in action!*
https://www.loom.com/share/829b90d831ea441ba2db6bea724af210

## Relevant Links <!-- Link to all the things that are required outside of the ones that have their own section -->
- [Check out the deployed site](#https://hiphoppizzaandwangspos.netlify.app/)
- [Wireframes](#https://www.figma.com/file/4y3EZddALuBR3ouSEM57Np/MVP?node-id=0%3A1)
- [Project Board](#https://github.com/orgs/nss-evening-cohort-24/projects/9/views/1)
- [Check out our technical flow!](#https://user-images.githubusercontent.com/111099613/267167644-5f3dc7a1-576e-4323-9791-c46e2c5ece6c.png)
- [ERD? Sounds good to me!](#https://user-images.githubusercontent.com/97858910/266849569-ca8b3488-c77c-4248-9c43-0f2b08b2fef5.png)

## Code Snippet <!-- OPTIONAL, but doesn't hurt -->
Look, ma, I can use array methods! - REDUCE
```
  const total = res.orderItems.reduce((acc, curr) => acc + curr.price, 0);
```


## Project Screenshots <!-- These can be inside of your project. Look at the repos from class and see how the images are included in the readme -->
<img width="auto" alt="Your Alt" src="your-link.png">

## Contributors
- [Shari Ebach](https://github.com/GitEbachS)
- [Ryan Shore](https://github.com/mshorecode)
- [Keana Cobarde](https://github.com/keanacobarde)

```

**Comment by keanacobarde** [comment]
FOUND README TEMPLATE
```
# This is the name of my Project  [![Netlify Status](https://api.netlify.com/api/v1/badges/4ab7e730-7ed3-4cfd-a988-66195e79a991/deploy-status)](https://app.netlify.com/sites/drt-sortinghat/deploys)
<!-- update the netlify badge above with your own badge that you can find at netlify under settings/general#status-badges -->

Here I am putting an overview of what my project is about. It comes below the name of my project so that others can read what it is about and get more details.

[View App](#your-link)

## Get Started <!-- OPTIONAL, but doesn't hurt -->
PLACE CODE SNIPPET HERE

## About the User <!-- This is a scaled down user persona -->
- The ideal user for this application is a teacher
- They have students in their classrooms that they would like to put into random groups and they have a love and passion of Harry Potter
- The problem this app solves for them is it allows them to get their students involved and excited about being in random groups. The students have felt that the groups have not been so random and based on favorites.

## Features <!-- List your app features using bullets! Do NOT use a paragraph. No one will read that! -->
- When a new student is added an object should be created and that object should be pushed into an array of students that then prints to the DOM.
- House Colors: The color of the student's card changes depending on which house they were sorted.
- Card Ordering: Sort the student cards by some criteria (i.e. alphabetically by name, by house)
- Voldermort's Army: Create a separate container of cards that hold the cards for students that have been expelled. These should be styled differently from Hogwarts students.

## Video Walkthrough of APP NAME <!-- A loom link is sufficient -->
https://www.loom.com/share/829b90d831ea441ba2db6bea724af210

## Relevant Links <!-- Link to all the things that are required outside of the ones that have their own section -->
- [Check out the deployed site](#your-link)
- [Wireframes](#your-link)
- [Project Board](#your-link)

## Code Snippet <!-- OPTIONAL, but doesn't hurt -->
PLACE CODE SNIPPET HERE

## Project Screenshots <!-- These can be inside of your project. Look at the repos from class and see how the images are included in the readme -->
<img width="1148" alt="Your Alt" src="your-link.png">

## Contributors
- [YOUR NAME](https://github.com/your-github-url)
```

**Comment by keanacobarde** [comment]
CLOSED PER PR #52 - CLOSING TICKET.
